engine/interpreter-dsl-test/src/test/java/org/enso/interpreter/dsl/test/InliningBuiltinsTest.java

```diff
@@ -8,141 +8,131 @@
 import com.oracle.truffle.api.nodes.RootNode;
 import org.enso.interpreter.node.InlineableNode;
 import org.enso.interpreter.runtime.callable.function.Function;
-import org.enso.test.utils.ContextUtils;
+import org.enso.test.utils.ContextRule;
+import org.junit.ClassRule;
 import org.junit.Test;
 
 public class InliningBuiltinsTest {
+  @ClassRule public static final ContextRule ctxRule = ContextRule.createDefault();
```

Comment:
Don't repeat yourself violation:
- `@ClassRule` - "rule" used for the first time
- `ContextRule` - "rule" again
- `ctxRule` - 3rd usage of "rule"

A little bit too much "rules" from my perspective.

```suggestion
  @ClassRule public static final Kontext ctx = Kontext.create();
```

PS: I am making `Kontext` name up. I don't want to suggest the right name for this class (yet).

Reply:
Named as `ContextUtils` in 952b6752b2254a65fe63107c919021824066cb6c